src/main/java/org/quartzpowered/common/util/CryptoUtil.java

```diff
@@ -41,7 +41,8 @@
 @Singleton
 public class CryptoUtil {
 
-    @Inject private Logger logger;
+    @Inject
+    private Logger logger;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.